docs/source/pages/Visualization.rst

```diff
@@ -10,6 +10,11 @@ Matplotlib_. Two different approaches are suggested.
 
 :ref:`plt.contourf`
 
+.. warning::
```

Comment:
I think it might be good to state the *reason* that they need to be transposed as that may make this a bit clearer?

Reply:
Thanks, this is a good point. I extended the warning to add a detailed explanation of what's going on.